tests/test_ujson.py

```diff
@@ -865,6 +866,29 @@ def default(value):
         ujson.dumps(unjsonable_obj, default=default)
 
 
+def test_dump_huge_indent():
+    ujson.encode({"a": True}, indent=65539)
+
+
+def test_dump_long_string():
+    ujson.dumps(["aaaa", "\x00" * 10921])
```

Comment:
It's probably a good idea to test broader than this, like light fuzz tests with slightly varying indentation depths and string lengths. That way, future changes that shift these things in minor ways (e.g. spaces after key and item separators by default) wouldn't render these tests immediately irrelevant.